Formula/cash-cli.rb

```diff
@@ -0,0 +1,19 @@
+require "language/node"
+
+class CashCli < Formula
+  desc "Exchange Currency Rates using your terminal!"
+  homepage "https://github.com/xxczaki/cash-cli"
+  url "https://registry.npmjs.org/cash-cli/-/cash-cli-2.0.6.tgz"
+  sha256 "a6d7e848a4ec51bb3bc51aa5317eb661e55d8f9130a368664580d3da11f2d335"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    raise "Test not implemented."
```

Comment:
You mean I need to add tests?

Reply:
Yes please!

Reply:
@DomT4  Would something on the following lines be an acceptable test?
```rb
test do
  assert_equal shell_output("#{bin}/cash 100 INR USD GBP".split("\n").last),
                            "\n\nConversion of INR 100\n"
end
```

Reply:
@rakkesh Yes, thank you!

Reply:
If we use `assert_match` rather than `assert_equal` we might be able to get away with:
```ruby
assert_match "Conversion of INR 100", shell_output("#{bin}/cash 100 INR USD GBP")
```
possibly?

Reply:
Done 👍